addons/marketing_card_event/models/mailing_mailing.py

```diff
@@ -0,0 +1,25 @@
+from ast import literal_eval
+
+from odoo import api, fields, models
+
+
+class MailingMailing(models.Model):
+    _inherit = 'mailing.mailing'
+
+    mailing_domain = fields.Char(compute='_compute_mailing_domain', readonly=False, store=True)
+    event_id = fields.Many2one('event.event', string='Related Event')
```

Comment:
This is never present in the user interface? It's basically a technical field? :) (used within its the default value in the action?)

Reply:
Yes

It could be made to work with context values if this were a wizard but as it has to work when the user visits the mailing later on I don't think there's really an alternative :/